__tests__/app.test.js

```diff
@@ -288,4 +288,32 @@ test('should return an article by its id', () => {
       expect(body).toHaveProperty('article')
       expect(body.article).toHaveProperty('article_id', 1)
     })
+})
+
+describe("DELETE /api/comments/:comment_id", () => {
+  test("should delete a comment by comment_id", () => {
+    const commentId = 1;
```

Comment:
Seems unnecessary to store a variable of the number 1 - especially as you only use it once!